src/g3log/logworker.hpp

```diff
@@ -101,6 +102,12 @@ namespace g3 {
          return std2::make_unique<SinkHandle<T>> (sink);
       }
 
+      /// Removes a sink
+      /// @param
+      template<typename T>
+      void removeSink(std::unique_ptr<SinkHandle<T>> sink_handle) {

```

Comment:
So the `SinkHandle<T>` is received as a client as a `unique_ptr`. Since it is "removed" here it makes sense to "give it up" (never to be seen again). Is that why you chose to send it as a `unique_ptr`?

I.e. the call would be 

```logworker->removeSink(std::move(myHandle));```